src/main/java/com/epam/page/object/generator/adapter/JavaPoetAdapter.java

```diff
@@ -117,7 +116,8 @@ private void createPageFields(String packageName, List<FieldSpec> pageFields, St
     }
 
     private ClassName getPageClassName(String packageName, String pageClassName) {
-        return ClassName.get(packageName.substring(0,packageName.length()-5) + ".page", pageClassName);
+        return ClassName
+            .get(packageName.substring(0, packageName.length() - 5) + ".page", pageClassName);
```

Comment:
What does this magic number mean?

Reply:
I have no idea, i just applied our TAI codestyle to our project